### DISCUSSION TITLE
Unsupported engine installing quasar cli

### DISCUSSION THREAD

**Discussion by joweste** [quasarframework/quasar#15959] [General - Components / Directives / etc] [answered]
Hi, I am a newbie in quasar.
I need to use quasar with vue2.
So, I have followed the instructions from quasar docs and I make the install.  But I have the next warnings.
If possible, I´d like to use my currenty node version and don´t use nvm, because I have several projects. I´d prefer don´t have to change the node version all the times in my other projects.
  
npm install -g @quasar/cli
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'execa@7.1.1',
npm WARN EBADENGINE   required: { node: '^14.18.0 || ^16.14.0 || >=18.0.0' },
npm WARN EBADENGINE   current: { node: 'v16.9.1', npm: '7.24.1' }
npm WARN EBADENGINE }

added 220 packages, and audited 221 packages in 16s

73 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

**Comment by rstoenescu** [comment, marked as answer]
Install only even Node.js version number (14/16/18/....)
These are stable LTS releases. The uneven versions (15/17/19/...) are not reliable.

**Reply by yusufkandemir** [reply]
In this case, they are using v16, but v16.9.1, which is old.

@joweste you can update Node from v16.9.1 to v16.20.0 to solve this problem and get a more reliable Node version.
